### PR TITLE
Alpha autoupdater

### LINKED ISSUE #405: [Feature Request]: Update the current version of `NetSparkle`
Update the current version of 'NetSparkle' to fix bugs since its integration

### PR DESCRIPTION
**Pull request by Wagnerp**
* Closes #405 & #414

### CODE REVIEW

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Software.Updater/Auto Updater/UX/DownloadUpdateDialog.cs [hunk 1]
```diff
@@ -0,0 +1,331 @@
+﻿using System;
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Software.Updater/Auto Updater/UX/DownloadUpdateDialog.cs]: New file needs the license agreement

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Software.Updater/Auto Updater/UX/DownloadUpdateDialog.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Software.Updater/Auto Updater/UX/DownloadUpdateDialog.cs]: Done :)

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Software.Updater/Auto Updater/Translations/AutoUpdaterStrings.cs [hunk 2]
```diff
@@ -1,4 +1,31 @@
-﻿namespace Krypton.Toolkit.Suite.Extended.Software.Updater
+﻿#region MIT License
+/*
+ * MIT License
+ *
+ * Copyright (c) 2017 - 2023 Krypton Suite
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Software.Updater/Auto Updater/Translations/AutoUpdaterStrings.cs:5]: Wrong start date for new files

Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Software.Updater/Auto Updater/Translations/AutoUpdaterStrings.cs]: Fixed

**Pull request merged**